src/NServiceBus.Persistence.Sql.TransactionalSession/SqlPersistenceTransactionalSessionExtensions.cs

```diff
@@ -0,0 +1,21 @@
+namespace NServiceBus.TransactionalSession
+{
+    using Configuration.AdvancedExtensibility;
+    using Features;
+
+    /// <summary>
+    /// Enables the transactional session feature.
+    /// </summary>
+    public static class SqlPersistenceTransactionalSessionExtensions
+    {
+        /// <summary>
+        /// Enables transactional session for this endpoint.
+        /// </summary>
+        public static PersistenceExtensions<SqlPersistence> EnableTransactionalSession(
```

Comment:
Should we add `Open` extension method to enable scenarios where no options are passed by the user?

Reply:
Same comment as with RavenDB.